### PR TITLE
feat(frontend): Show warning and disable send if no utxos

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

User can't send tokens if there are not available utxos.

Show a warning if the endpoint to select utxos and the fee returns no utxos.

# Changes

* Rename `BtcSendHasPendingTransactions` to `BtcSendWarnings`.
* Add prop `utxosFee` to `BtcSendWarnings` and use it to add a warning.
* Add new i18n key.

# Tests

Tested locally.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2024-10-14 at 10 47 56

**Comment by lmuntaner** [comment]
@peterpeterparker @AntonioVentilii-DFINITY please review

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/btc/components/send/BtcSendWarnings.svelte [hunk 1]
```diff
@@ -16,3 +18,9 @@
 		<WarningBanner>{$i18n.send.error.no_pending_bitcoin_transaction}</WarningBanner>
 	</div>
 {/if}
+
+{#if utxosFee?.utxos.length === 0}
+	<div in:fade>
+		<WarningBanner>{$i18n.send.info.no_available_utxos}</WarningBanner>
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply src/frontend/src/btc/components/send/BtcSendWarnings.svelte:24]: I think that users won't read more than one sentence at a time. Better having two warnings separately.

If we realize it bothers users too much. We might want to try to be smarter and show only one warning at a time in those cases.

**Pull request merged**